bugbug/similarity.py

```diff
@@ -93,6 +94,10 @@ def __init__(self, cleanup_urls=True, nltk_tokenizer=False):
             self.cleanup_functions.append(feature_cleanup.url())
 
         self.nltk_tokenizer = nltk_tokenizer
+        self.threshold = threshold
+
+    def get_threshold(self):
+        return self.threshold
```

Comment:
You can remove this function, user of the class can just do model.threshold.

Also, rename it as `confidence_threshold`